include/iwinfo.h

```diff
@@ -224,7 +238,9 @@ struct iwinfo_rate_entry {
 	uint8_t is_he:1;
 	uint8_t he_gi;
 	uint8_t he_dcm;
-	uint8_t mhz;
```

Comment:
This change breaks the ABI which I'd ideally like to avoid. If I am not mistaken, there should be two bytes of padding at the end of the struct, so maybe place the high mhz bits after the `mss` field. Also put the `is_eht` bitfield member directly after `is_he` to ensure that it ends up in the same byte.

Updated structure layout could look like:

```
struct iwinfo_rate_entry {
	uint32_t rate;
	int8_t mcs;
	uint8_t is_40mhz:1;
	uint8_t is_short_gi:1;
	uint8_t is_ht:1;
	uint8_t is_vht:1;
	uint8_t is_he:1;
	uint8_t is_eht:1;
	uint8_t he_gi;
	uint8_t he_dcm;
	uint8_t mhz_lo;
	uint8_t nss;
        uint8_t mhz_hi;
        uint8_t eht_gi;    
};
```